changelogs/fragments/66921-sanity-state-get.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - 'ansible-test - Add a test to prevent ``state=get``'
```

Comment:
One possibility would also to be update the previous PR's changelog, since these kind of belong together.